pkg/virt-launcher/notify-client/client.go

```diff
@@ -279,6 +279,27 @@ func eventCallback(c cli.Connection, domain *api.Domain, libvirtEvent libvirtEve
 	}
 }
 
+var updateEvents = updateEventsClosure()
+func updateEventsClosure() (f func(event watch.Event, domain *api.Domain, events chan watch.Event)) {
+	var firstAddEvent bool = true
+	var firstDeleteEvent bool = true
+
+	f = func(event watch.Event, domain *api.Domain, events chan watch.Event) {
+		if event.Type == watch.Added && firstAddEvent {
+			firstAddEvent = false
+			events <- event
+			log.Log.Infof("HHH sent ADD")
+		} else if event.Type == watch.Modified && domain.ObjectMeta.DeletionTimestamp != nil && firstDeleteEvent {
```

Comment:
i assume there is just one delete event interesting, (the logic before also just used the first in case there were couple)